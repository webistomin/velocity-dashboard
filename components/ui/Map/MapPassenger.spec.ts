import { shallowMount } from '@vue/test-utils';
import MapPassenger from './MapPassenger';

describe('MapPassenger', () => {
  const factory = (propsData = {}, options = {}) => {
    return shallowMount(MapPassenger, {
      propsData: {
        ...propsData,
      },
      ...options,
    });
  };

  it('Matches snapshot', () => {
    const wrapper = factory().html();
    expect(wrapper).toMatchSnapshot();
  });
});
