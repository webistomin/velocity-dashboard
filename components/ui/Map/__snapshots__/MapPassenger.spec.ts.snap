// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapPassenger Matches snapshot 1`] = `
"<div class=\\"map__passenger\\">
  <div class=\\"map__passenger-col map__passenger-col_left\\">
    <div class=\\"map__passenger-heading\\">
      <vuecomponent-stub image=\\"/img/avatar.png\\" alt=\\"User\\" size=\\"m\\" class=\\"map__passenger-avatar\\"></vuecomponent-stub>
      <div class=\\"map__passenger-info\\"><strong class=\\"map__passenger-name\\">Nora M. Buchanan</strong><span class=\\"map__passenger-count\\">4 interactions</span></div>
    </div>
    <ul class=\\"map__passenger-list list\\">
      <li class=\\"map__passenger-item list-item\\"><strong class=\\"map__passenger-caption caption\\">Email</strong><span class=\\"map__passenger-value\\">invision@invisionapp.com</span></li>
      <li class=\\"map__passenger-item list-item\\"><strong class=\\"map__passenger-caption caption\\">Phone</strong><span class=\\"map__passenger-value\\">+144–3412–4422</span></li>
      <li class=\\"map__passenger-item list-item\\"><strong class=\\"map__passenger-caption caption\\">Location</strong><span class=\\"map__passenger-value\\">New York, NY</span></li>
    </ul>
  </div>
  <div class=\\"map__passenger-col map__passenger-col_right\\">
    <ul class=\\"map__passenger-payments list\\">
      <li tabindex=\\"0\\" class=\\"map__passenger-payment list-item\\"><img src=\\"/img/visa.svg\\" alt=\\"Visa\\" class=\\"map__passenger-payment-logo image\\">
        <div class=\\"map__passenger-payment-details\\">1111 1111 1111 1111</div>
      </li>
      <li tabindex=\\"0\\" class=\\"map__passenger-payment list-item\\"><img src=\\"/img/visa.svg\\" alt=\\"Visa\\" class=\\"map__passenger-payment-logo image\\">
        <div class=\\"map__passenger-payment-details\\">1111 1111 1111 1111</div>
      </li>
      <li tabindex=\\"0\\" class=\\"map__passenger-payment list-item\\"><img src=\\"/img/visa.svg\\" alt=\\"Visa\\" class=\\"map__passenger-payment-logo image\\">
        <div class=\\"map__passenger-payment-details\\">1111 1111 1111 1111</div>
      </li>
      <li tabindex=\\"0\\" class=\\"map__passenger-payment list-item\\"><img src=\\"/img/visa.svg\\" alt=\\"Visa\\" class=\\"map__passenger-payment-logo image\\">
        <div class=\\"map__passenger-payment-details\\">1111 1111 1111 1111</div>
      </li>
    </ul>
  </div>
</div>"
`;
